### PR TITLE
updating readme and rust folder readme to include altrios-style test/version badges

### PR DESCRIPTION
**Pull request by robinsteuteville**
@calbaker I added the readme badges. Didn't change anything other than the altrios references, so, if the python compatibilities or release version need to be different let me know and I can change those. Added the badges to both the overall readme and the rust/readme files.

### CODE REVIEW

**Comment by robinsteuteville** [comment]
@calbaker I added the readme badges. Didn't change anything other than the altrios references, so, if the python compatibilities or release version need to be different let me know and I can change those. Added the badges to both the overall readme and the rust/readme files.

File: README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 ![FASTSim Logo](https://www.nrel.gov/transportation/assets/images/icon-fastsim.jpg)
 
+[![Tests](https://github.com/NREL/fastsim/actions/workflows/tests.yaml/badge.svg)](https://github.com/NREL/fastsim/actions/workflows/tests.yaml) [![wheels](https://github.com/NREL/fastsim/actions/workflows/wheels.yaml/badge.svg)](https://github.com/NREL/fastsim/actions/workflows/wheels.yaml?event=release) ![Release](https://img.shields.io/badge/release-v0.1.0-blue) ![Python](https://img.shields.io/badge/python-3.9%20%7C%203.10-blue) [![Documentation](https://img.shields.io/badge/documentation-custom-blue.svg)](https://nrel.github.io/fastsim/) [![GitHub](https://img.shields.io/badge/GitHub-fastsim-blue.svg)](https://github.com/NREL/fastsim)
```
Comment (calbaker) [inline README.md]: @robinsteuteville , could you remove the `Release` badge?  I've found taht to be a maintenance hassle in ALTRIOS?

File: README.md [hunk 2]
```diff
@@ -1,5 +1,7 @@
...
+
 # Description
 This is the python/rust flavor of [NREL's FASTSim<sup>TM</sup>](https://www.nrel.gov/transportation/fastsim.html), which is based on the original Excel implementation. Effort will be made to keep the core methodology between this software and the Excel flavor in line with one another.
```
Comment (calbaker) [inline README.md:6]: @robinsteuteville, is this superscript?  Slick!

File: rust/README.md [hunk 3]
```diff
@@ -1,4 +1,6 @@
 # Crate Architecture
+[![Tests](https://github.com/NREL/fastsim/actions/workflows/tests.yaml/badge.svg)](https://github.com/NREL/fastsim/actions/workflows/tests.yaml) [![wheels](https://github.com/NREL/fastsim/actions/workflows/wheels.yaml/badge.svg)](https://github.com/NREL/fastsim/actions/workflows/wheels.yaml?event=release) ![Release](https://img.shields.io/badge/release-v0.1.0-blue) ![Python](https://img.shields.io/badge/python-3.9%20%7C%203.10-blue) [![Documentation](https://img.shields.io/badge/documentation-custom-blue.svg)](https://nrel.github.io/fastsim/) [![GitHub](https://img.shields.io/badge/GitHub-fastsim-blue.svg)](https://github.com/NREL/fastsim)
```
Comment (calbaker) [inline rust/README.md]: @robinsteuteville , could you remove the `Release` badge?

**Review by calbaker — APPROVED** [review]

**Pull request merged**